### DISCUSSION TITLE
Get error for alpyne attribute root path.

### DISCUSSION THREAD

**Discussion by gjones1911** [the-anylogic-company/Alpyne#34] [Q&A]
Hello all, 
I am attempting to run an exported model in python using Alpyne. I am using Python 3.10.5 on a windows machine using VScode. When I attempt to run the example Stock Manager model, I get the following error: 
    return alpyne._ROOT_PATH.joinpath("resources")
AttributeError: module 'alpyne' has no attribute '_ROOT_PATH' 

Has anyone run into this issue?

**Comment by listen-codeing** [comment]
i also get the error. but i cannot solve the issue.

**Reply by t-wolfeadam** [reply]
Hi, I'll be releasing a new version shortly that should fix these issues
